api/v01/api.rb

```diff
@@ -67,10 +77,76 @@ def count_incr(operation, options)
         def set_locale
           I18n.locale = env.http_accept_language.compatible_language_from(I18n.available_locales.map(&:to_s)) || I18n.default_locale
         end
+
+        def count_base_key(operation, period = :daily)
```

Comment:
why count functions are different from this commit https://github.com/Mapotempo/optimizer-api/pull/45/commits/d241e770f91dcb6b59ab487b5e5c7c5937475ac3 ?